crates/revm/src/executor.rs

```diff
@@ -495,7 +495,7 @@ pub fn commit_state_changes<DB>(
                 }
             };
 
-            cached_account.account_state = if account.storage_cleared {
+            cached_account.account_state = if account.is_created() {
```

Comment:
is this right?

Reply:
Went off of these here https://github.com/bluealloy/revm/pull/477/files#diff-1038cfcfe4447ae4de4b2108579796fe5a74475846a8581378b5e770556d628cR191

(`newly_created` was renamed to `created`)